repair/row_level.cc

```diff
@@ -399,7 +399,7 @@ class repair_writer_impl : public repair_writer::impl {
     reader_permit _permit;
     uint64_t _estimated_partitions;
     std::optional<future<>> _writer_done;
-    mutation_fragment_queue _mq;
+    lw_shared_ptr<mutation_fragment_queue> _mq;
```

Comment:
What I meant to suggest is to change `mutation_fragment_queue` to store its `_impl` as a shared pointer, so that `mutation_fragment_queue` can be passed around as values, and we don't need stable addresses for them.

Reply:
That makes more sense, let me fix it before merging.